src/BlazorStrap.Extensions.FluentValidation/FluentValidation.cs

```diff
@@ -4,19 +4,41 @@
 using FluentValidation;
 using Microsoft.AspNetCore.Components;
 using Microsoft.AspNetCore.Components.Forms;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace BlazorStrap.Extensions.FluentValidation
 {
-    public class FluentValidator<TValidator> : ComponentBase where TValidator : IValidator, new()
+    public class FluentValidator<TValidator> : BaseFluentValidator
+        where TValidator : IValidator, new()
+    {
+        protected override IValidator CreateValidator()
+        {
+            return new TValidator();
+        }
+    }
+
+    public class InjectFluentValidator : BaseFluentValidator
```

Comment:
Favor FluentValidatorInjected vs InjectFluentValidator

Reply:
OK I've made the change